doc/operations/configuration.rst

```diff
@@ -0,0 +1,235 @@
+
+.. _start configuration:
+
+``start``
+~~~~~~~~~
+
+.. note:: This only needs to be set when the node started in ``start`` mode.
+
+- ``constitution_files``: List of constitution files. These typically include ``actions.js``, ``validate.js``, ``resolve.js`` and ``apply.js``.
+
+- ``members``: List of initial consortium members files, including identity certificates, public encryption keys and member data files.
+
+.. note:: Common examples:
+
+    - A recovery member with member data: ``{"certificate_file": "member_cert.pem", "encryption_public_key_file": "member_enc_pubk.pem", "data_json_file": "member_data.json"}``
+    - A recovery member with no member data: ``{"certificate_file": "member_cert.pem", "encryption_public_key_file": "member_enc_pubk.pem"}``
+    - A non-recovery member with member data: ``{"certificate_file": "member_cert.pem", "data_json_file": "member_data.json"}``
+    - A non-recovery member with no member data: ``{"certificate_file": "member_cert.pem"}``
+
+- ``service_configuration``: Initial service configuration, including:
+
+    - ``max_allowed_node_cert_validity_days``: The maximum number of days allowed for node certificate validity period. Default value: ``365`` days. TODO: Rename
+    - ``recovery_threshold``. Note that if the recovery threshold is set to ``0``, it is automatically set to the number of recovery members specified in ``members``.
+    - ``reconfiguration_type``. The type of reconfiguration for new nodes. Default value: ``OneTransaction``.
+
+Example:
+
+.. code-block:: json
+
+    "start": {
+        "constitution_files": ["actions.js", "validate.js", "resolve.js", "apply.js"],
+        "members": [
+            {"certificate_file": "member0_cert.pem", "data_json_file": null, "encryption_public_key_file": "member0_enc_pubk.pem"},
+            {"certificate_file": "member1_cert.pem", "data_json_file": "member1_data.json", "encryption_public_key_file": null},
+            {"certificate_file": "member2_cert.pem", "data_json_file": null, "encryption_public_key_file": "member2_enc_pubk.pem"}
+        ],
+        "service_configuration":
+        {
+            "recovery_threshold": 0,
+            "max_allowed_node_cert_validity_days": 365,
+            "reconfiguration_type": "OneTransaction"
+        }
+    }
+
+.. _join configuration:
+
+``join``
+~~~~~~~~
+
+.. note:: This only needs to be set when the node is started in ``join`` mode.
+
+- ``target_rpc_address``: Address (hostname and port) of a node of the existing service to join.
+- ``join_timer_ms``: Interval (ms) at which the node sends join requests to the existing network. Default value: ``1000`` ms.
+
+Example:
+
+.. code-block:: json
+
+    "join": {
+        "join_timer_ms": 1000,
+        "target_rpc_address": {"hostname": "127.0.0.1", "port": "8080"}
+    }
+
+``ledger``
+~~~~~~~~~~
+
+- ``ledger_dir``: Path to main ledger directory. Default value: ``ledger``.
```

Comment:
`ledger` feels redundant here, perhaps `append_directory` and `read_only_directories` (and `chunk_size_bytes`?)

Reply:
`chunk_size` as the unit will be in value string.